src/test/java/com/woowacourse/pelotonbackend/member/infra/LoginMemberArgumentResolverTest.java

```diff
@@ -58,11 +59,36 @@ void resolveArgumentTest() {
         assertThat(memberResponse).isEqualToComparingFieldByField(createWithId(ID));
     }
 
+    @DisplayName("해당 kakaoId와 매칭되는 회원정보가 존재하지 않을 경우 MemberNotFoundException으로 예외처리한다.")
+    @Test
+    void resolveArgumentTest2() {
+        final long kakaoId = 1L;
+        servletWebRequest.setAttribute("loginMemberKakaoId", String.valueOf(kakaoId), SCOPE_REQUEST);
+        given(memberService.findByKakaoId(kakaoId))
+            .willThrow(new MemberNotFoundException(String.format("Member(member kakaoId = %d not exist)", kakaoId)));
+
+        assertThatThrownBy(
+            () -> loginMemberArgumentResolver.resolveArgument(methodParameter, null, servletWebRequest, null))
+            .isInstanceOf(MemberNotFoundException.class)
+            .hasMessageContaining("Member(member kakaoId = %d not exist)", kakaoId);
```

Comment:
현재 제 브랜치에서 NotFoundException의 메시지들의 아래의 포맷을 통일하는 중입니다. 더 좋은 메시지가 있다면 제안해주시거나 변경해주세요~ 😄 
```suggestion
            .hasMessageContaining("Member(member kakaoId = %d) does not exist)", kakaoId);
```